README.md

```diff
@@ -0,0 +1,74 @@
+khealth
+=====
+
+[![Docker Repository on Quay.io](https://quay.io/repository/coreos/khealth/status "Docker Repository on Quay.io")](https://quay.io/repository/coreos/khealth)
+
+**khealth** is a kubernetes cluster monitoring suite. Routines are defined which exercise various kubernetes subsystems. Collectors receive status update events from Routines and compute current state.
+
+## Quick start
+
+If you have a kubernetes cluster, you can deploy khealth.
+
+```sh
+cd khealth/
+kubectl create -f ./contrib/k8s/khealth-ns.yaml
+kubectl create -f ./contrib/k8s/khealth-{rc,service}.yaml
+```
+
+This will create a nodeport service which exposes the following status endpoints.
+
+| Command  | NodePort |
+| ------------- |:-------------:|
+| rcscheduler  | 31337 |
+
+## Architecture
+
+A module is defined as a single entrypoint command which makes use of a set of routines and a single collector to expose a metric set via the collector's status endpoint.
```

Comment:
A khealth *Module* is a single command that executes a set of Routines and a single Collector. The Collector gathers data from the Routines and exposes metrics on its status endpoint.